website/src/pages/tutorial/basic/_meta.ts

```diff
@@ -10,5 +10,6 @@ export default {
   '08-graph-relations': 'Graph Relations',
   '09-error-handling': 'Error Handling',
   '10-filtering-and-pagination': 'Filtering and Pagination',
-  '11-summary': 'Summary',
+  '11-setting-up-graphql-code-generator': 'Setting up GraphQL Code Generator',
```

Comment:
I'm adding codegen tutorial at the end instead of the start for a few reasons:
- The first few chapters are great as a quick "Get Started" guide to help users understand how to set up a GraphQL Yoga server
- Codegen is an improvement to the workflow, which is highlighted in the tutorial